config/upstart.example/tfb.conf

```diff
@@ -0,0 +1,18 @@
+# /etc/init/tfb.conf
+env TFB_REPOPARENT="/private"
+env TFB_REPONAME="FrameworkBenchmarks"
+env TFB_REPOURI="git@github.com:ashawnbandy-te-tfb/FrameworkBenchmarks.git"
```

Comment:
Should this be the master repo URI? Or would you rather restrict it to your repo until we have all of the kinks ironed out?